NitroxClient/MonoBehaviours/Gui/MainMenu/JoinServer.cs

```diff
@@ -115,5 +162,67 @@ private void DoJoinServerWindow(int windowId)
                 shouldFocus = false;
             }
         }
+
+        private void startMultiplayerClient()
+        {
+            if (multiplayerClient == null)
+            {
+                multiplayerClient = new GameObject();
+                multiplayerClient.AddComponent<Multiplayer>();
```

Comment:
This was honestly a hack. For some reason, that patch must have stopped working for me because Multiplayer was not starting. I'll dig into this further to see what I broke because that makes me very uncomfortable.

Reply:
Well tbh, I don't like patching the memory if it isn't necessary. If you solution works 100% I think it's an improvement.

Reply:
I've been digging into this more and I have no clue how I broke this. It looks like the call to the static Prefix member is never called on that patch but I don't understand why.

I'll need to wait until I get home to dig more. Which sucks because I really don't like what I'm doing at work today -_-

I'll keep the solution I've rolled if possible. I just really want to understand what I changed to make this play out the way it did...

Reply:
We killed the patch as suggested. Thanks Measuring!